Remember The Date/View Controllers/ZenHelpViewController.m

```diff
@@ -95,6 +96,24 @@ -(void) setupSupportInformation {
     }];
 }
 
+- (void) setupSDK {
+    static dispatch_once_t onceToken;
```

Comment:
I also removed the dispatch once block and just use the same bool to guard the init.